src/Schema/Events/Search.ts

```diff
@@ -80,6 +84,36 @@ export interface SelectedItemFromSearch {
  */
 export interface SearchedWithNoResults {
   action: ActionType.searchedWithNoResults
+  activePill: PillType
+  context_module: ContextModule
+  context_owner_type: PageOwnerType
+  context_owner_id?: string
+  context_owner_slug?: string
+  destination_owner_type?: PageOwnerType
+  query: string
+}
+
+/**
+ * A user selects search pill
+ *
+ * This schema describes events sent to Segment from [[tappedNavigationTab]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedNavigationTab",
+ *    activePill: "artwork"
+ *    context_module: "header",
+ *    context_owner_type: "home",
+ *    destination_owner_type: "search",
+ *    query: "andhuwe"
+ *    pill: "artwork"
+ *  }
+ * ```
+ */
+export interface SelectedPillFromSearch {
```

Comment:
this event is duplicated by [an existing one](https://github.com/artsy/cohesion/blob/acf10eddd3f18226704c5688da5fd3bbe4c8ced4/src/Schema/Events/Tap.ts#L559-L567)